src/ec/packages/ec-component-timeline/timeline.html.twig

```diff
@@ -0,0 +1,101 @@
+{% spaceless %}
+
+{#
+Parameters:
+  - "toggle_collapsed": (string) (default: '')
+  - "toggle_expanded": (string) (default: '')
+  - "hide" (object): (default: undefined)
+    - "from": (integer) (default: undefined) Item index after which to start hiding timeline items
+    - "to": (integer) (default: undefined) Item index after which to resume displaying timeline items
+  - "items" (array) (default: []):
+    - "id": (string) (default: '')
+    - "label": (string) (default: '')
+    - "title": (string) (default: '')
+    - "content": (block) (default: '')
+  - "icon_path" (string) (default: ''): file containing the svg icons
+  - "extra_classes" (string) (default: '')
+  - "extra_attributes" (array) (default: []): format:
+    - "name" (string) (default: '')
+    - "value" (string) (default: '')
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-timeline2' %}
+{% set _extra_attributes = '' %}
+{% set _items = items|default([]) %}
+{% set _icon_path = icon_path|default('') %}
+{% set _toggle_collapsed = toggleCollapsed|default('') %}
+{% set _toggle_expanded = toggleExpanded|default('') %}
+{% set _hide_from = hide.from|default(items|length) %}
+{% set _hide_to = hide.to|default(items|length) %}
+
+{# Internal logic - Process properties #}
+
+{% set _display_amount = items|length %}
+{% set _hide_amount = 0 %}
+{% set _bottom_amount = 0 %}
+
+{% if _hide_from > 0 %}
```

Comment:
Which means a negative `from` index is not supported yet (this feature could be added later if needed).

This is not a blocker ;)